activesupport/test/metadata/shared_metadata_tests.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module SharedMessageMetadataTests
+  def setup
+    @message = { "credit_card_no" => "5012-6784-9087-5678", "card_holder" => { "name" => "Donald" } }
+    @string_message = "address: #23, main street"
+    @array_message = ["credit_card_no: 5012-6748-9087-5678", { "card_holder" => "Donald", "issued_on" => Time.local(2017) }, 12345]
```

Comment:
`@string_message` and `@array_message` are only used in one test. Let's just inline them to that test (remember newlines too).